release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1739,6 +1743,34 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
+
+* Previously, there was an issue with Patternfly 4 where you could not select or change the log component under the master node's logs section once a selection was already made. With this update, when you change to the log component from master node's log section, refresh the page to reload the default options. (link:https://issues.redhat.com/browse/OCPBUGS-18727[*OCPBUGS-18727*])
+
+* Previously, an empty page was displayed when viewing route details on the *Metrics* tab of the *`alertmanager-main`* page. With this update, user privileges were updated so you can view the route details on the *Metrics* tab. (link:https://issues.redhat.com/browse/OCPBUGS-15021[*OCPBUGS-15021*])
+
+* Previously, {product-rosa} used custom branding and the favicon would disappear so no specific branding appeared when custom branding is in use. With this update, {product-rosa} branding is now part of the branding API. (link:https://issues.redhat.com/browse/OCPBUGS-14716[*OCPBUGS#14716*])
+
+* Previously, the {product-title} web console did not render the monitoring *Dashboard* page when a Proxy was expected, and as a result websocket connection failed. With this update, the web console also detects proxy settings from environment variables. (link:https://issues.redhat.com/browse/OCPBUGS-14550[*OCPBUGS-14550*])
+
+* Previously, if the `console.openshift.io/disable-operand delete: "true"` and `operator.openshift.io/uninstall-message: "some message"` annotations were used on an operator CSV, the uninstall instructions did not show up in the web console. With this update, the instructions to opt out of the installment are available. (link:https://issues.redhat.com/browse/OCPBUGS-13782[*OCPBUGS-13782*])
+
+* Previously, the size on the *PersistentVolumeClaims* namespace *Details* page was incorrect. With this update, the prometheus query on *PersistentVolumeClaims* namespace *Details* page includes the namespace label and the size is now correct. (link:https://issues.redhat.com//browse/OCPBUGS-13208[*OCPBUGS-13208*])
+
+* Previously, after customizing the routes for console and downloads, the downloads route did not update in the `ConsoleCLIDownloads` link and pointed to the default downloads route. With this update, the `ConsoleCLIDownloads` link updates when the custom downloads route is set. (link:https://issues.redhat.com/browse/OCPBUGS-12990[*OCPBUGS-12990*])
+
+* Previously, the print preview displayed incomplete topology information from the list view. With this update, a full list of resources is printed when they are longer than one page. (link:https://issues.redhat.com/browse/OCPBUGS-11219[*OCPBUGS-11219*])
+
+* Previously, dynamic plugins that proxy to services with longer response times timed out at 30 seconds with a 504 response. With this updatee, a 5 minute haproxy timeout annotation was added to the console route to match most browsers maximum timeout. (link:https://issues.redhat.com/browse/OCPBUGS-9917[*OCPBUGS-9917*])
```

Comment:
```suggestion
* Previously, dynamic plugins that proxy to services with longer response times timed out at 30 seconds with a 504 error message. With this update, a 5-minute HAProxy timeout annotation was added to the console route to match the maximum timeout of most browsers. (link:https://issues.redhat.com/browse/OCPBUGS-9917[*OCPBUGS-9917*])
```
https://docs.openshift.com/container-platform/3.6/architecture/networking/haproxy-router.html for style